app/config/index.js

```diff
@@ -42,6 +42,7 @@ const schema = Joi.object({
   env: Joi.string().valid('development', 'test', 'production').default(
     'development'
   ),
+  googleTagManagerKey: Joi.string().default('GTM-5HKTPLK'),
```

Comment:
Should there be a default set as this will effectively mean all development work would be using it? I wonder if local dev should only include it if there is an env var set?